### ISSUE TITLE
Show vault unlock menu when selecting "Vault is locked" in context menu

### ISSUE THREAD

**Issue by lazyc97** [bitwarden/browser#272]
I belive this is more convenient.

**Comment by kspearrin** [comment]
This cannot be done. A locked vault means there is no decryption key (which is why you have to type in your password to unlock).

**Comment by lazyc97** [comment]
I mean the "login" menu can be shown from clicking that with password input box focused, ready to login. Instead of having to move my mouse to toolbar and click the icon then type in password.

**Comment by kspearrin** [comment]
Sorry, but I do not understand what you are describing.

**Comment by lazyc97** [comment]
Maybe "unlock menu" would be more correct than "login menu".

I want the work flow like this:
1. Open context menu > Bitwarden > auto-fill
2. Found out I haven't logged in 😢 
3. Chose "Vault is locked", the "Verify Master Password" panel show up with "Master Password" box focused
4. Type my password without moving my mouse
5. Yay, I got in.

Currently after step 2, I have to move my mouse to the Bitwarden icon on the opposite corner of the screen to unlock.

**Comment by kspearrin** [comment]
Unfortunately the browser APIs do not allow us to open the popup window without you physically clicking the bitwarden icon button yourself.

**Comment by favadi** [comment]
Not sure I should file new feature request. Currently when vault is locked and I press Ctrl+Shift+L, nothing happens. It should be more convenient to open the login popup, right?

**Comment by kspearrin** [comment]
@favadi Would only be doable in Firefox. Open a new issue if this is what you desire. This issue can be marked resolved with #314

**Comment by favadi** [comment]
I created https://github.com/bitwarden/browser/issues/433.
